components/InventoryItemList/index.tsx

```diff
@@ -95,7 +97,7 @@ const headCells: readonly HeadCell[] = [
   },
   {
     id: 'quantity',
-    numeric: true,
+    numeric: false,
```

Comment:
Just putting a comment here, I believe I changed this to false so that all of the headers are aligned the same way. If it's set to true, then the header cell is not aligned with the row cell content. @RudraPatel2003 I think you set this up originally. Let me know what you all think.

Here's what it looks like when it's set to `true`
![image](https://user-images.githubusercontent.com/71574118/229692423-9268da68-5a64-48f4-827f-2cd4a5115441.png)

Reply:
I like it way better with it set equal to false. I think it is way more intuitive and easy to read. However, will that change the type of the Quantity field? Like will setting numeric equal to false not allow Quantity be numerical values, or does it just affect the display of the table?

Reply:
Great question, and one that I haven't thought about. I looked into it, and I think we're OK for a few reasons:
1. [This](https://codesandbox.io/s/keen-stallman-f7r9cz?file=/demo.tsx) example I pulled from MUI does filtering properly even if I set the `numeric` properties to `false`
2. (Currently) The `numeric` property is not used elsewhere
3. Our data is stored as a number, so any TS/JS sorting will know to treat them as numbers